packages/wasm/crate/src/planner.rs

```diff
@@ -482,6 +484,40 @@ pub async fn plan_transaction(
         ));
     }
 
+    for tpr::Spend { value, address } in request.spends {
```

Comment:
Do I understand this correctly: if a tx planner request has a `spend`, their intention is to take every note they have and put it in the ownership of another address?

Reply:
correct, we spend the entire SNR set, and set the change address to the receiver.